### PR TITLE
replaced dark mod button text with icon.

### PR DESCRIPTION
**Pull request by chikkibum**
@arpitghura hey i have solved this issue #77

i added `<i id="icon1" onclick="change('icon1')" onclick="toggleLightDarkMode()" class="bx bx-moon dark-btn" style="font-size: x-large;"></i>`  in practice.html.

practice

similarly, I added `<i id="icon1" onclick="change('icon1')" onclick="toggleLightDarkMode()" class="bx bx-moon dark-btn" style="font-size: x-large;"></i>` to about and profile page also.

Then i added js to change the icon from moon to sun using 
`function change (iconID){
    if(document.getElementById(iconID).className=="bx bx-moon dark-btn"){
      document.getElementById(iconID).className = "bx bx-sun dark-btn";
    }
else{
      document.getElementById(iconID).className = "bx bx-moon dark-btn";
    }
  }
`

icon

and called it whenever needed.

here is a video attached to show that its working:

https://user-images.githubusercontent.com/71315985/215354809-5e2c73e5-95be-4ba4-8fbb-fb2a110ff76b.mp4

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 <div class="github-icon">
                     <a href="https://github.com/arpitghura/typing-test" class="nav-item-link"><i class='bx bx-sm bxl-github'></i></a>
                 </div>
-                <button onclick="toggleLightDarkMode()" class="dark-btn">Dark Mode</button>     
+                <i id="icon1" onlick="change('icon1')" onclick="toggleLightDarkMode()" class="bx bx-moon dark-btn" style="font-size: x-large;"></i>
```
Comment (arpitghura) [inline about.html:31]: What is `onlick="change('icon1')"` ? I am not sure that onlick is an event listener, do you want to use `onclick`?

Comment (chikkibum) [reply about.html:31]: yea  i guess i made a typo

Comment (chikkibum) [reply about.html:31]: @arpitghura let me fix that

**Comment by chikkibum** [comment]
@arpitghura I checked and i by mistake i also added `onclick="change('icon1')"` in the `< i >` tag while I already used `change('icon1')` in dark.js file.

Fix: removed  `onclick="change('icon1')"` from the `< i >` tag.

**Review by arpitghura — APPROVED** [review]

**Pull request merged**